mvp/test/java/AlgorithmTest.java

```diff
@@ -59,19 +58,26 @@ void testRoundRobinNext() {
        testListExpected.add(new Server(5, "ip7"));
 
        algo.setServerList(testListExpected);
-       algo.RoundRobinNext();
-       assertThat( algo.RoundRobinNext(), testListExpected.get(0));
-
-
-
-
+       
+       //I expect RoundRobinNext() to return testListExpected[0]
+       
+       assertEquals(algo.RoundRobinNext(),testListExpected.get(0));
```

Comment:
you can write like this.
for(int i=0;i<list.size;i++)
assertEquals(algo.roundrobinnext,testlistexpect.get(i))